1.20/src/main/java/io/github/axolotlclient/config/screen/CreditsScreen.java

```diff
@@ -161,11 +161,11 @@ private void initCredits() {
 		credits.add(new Credit("YakisikliBaran", "Turkish Translation"));
 		credits.add(new Credit("TheKodeToad", "Contributor", "Motion Blur", "Freelook", "Zoom"));
 		credits.add(new Credit("DragonEggBedrockBreaking", "Bugfixing", "Inspiration of new Features"));
+		credits.add(new Credit("DarkKronicle", "Bedwars Overlay", "Author of KronHUD, the best HUD mod!"));
```

Comment:
I think darkkronicle should be above dragoneggbedrockbreaking as while they haven't directly contributed, the code you have used from him is pretty major :laughing:

Reply:
I don't think that this list was sorted in any way, rather just appended when someone new contributed. If this is important to you or anyone else, it can be changed.

Reply:
You moved DarkKronicle

Reply:
from the "Other people" list to the "Contributors" list (i think)

Reply:
ah... i wonder if this makes sense... but i understand

Reply:
It might be clearer on the API rewrite branch where the Credits data has been put into its own class